src/KPM.jl

```diff
@@ -1,10 +1,14 @@
 #######################################################################
 # Kernel Polynomial Method : momenta
 #######################################################################
+using Base.Threads
+import Base.Threads.@spawn
```

Comment:
You don't seem to use `@spawn`, this can be removed. Also, could remove `using Base.Threads` just by writing `Threads.@threads` instead of `@threads`